src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -54899,8 +54899,8 @@ INSERT INTO tab_container_parameter_rates VALUES(5011,'GENERAL','MoleculePropert
 INSERT INTO tab_container_parameter_rates VALUES(1678,'COMPOUND','DRUG','Enable supersaturation','CompoundCommon','Zero_Rate',0.0,1,1.0,1,NULL);
 INSERT INTO tab_container_parameter_rates VALUES(5015,'GENERAL','PROTEIN','Fraction expressed on tissue-side membrane of vascular endothelium','ExpressionParameters','PARAM_f_exp_vasc_tissue_side',0.0,1,1.0,1,NULL);
 INSERT INTO tab_container_parameter_rates VALUES(5015,'GENERAL','PROTEIN','Fraction expressed on plasma-side membrane of vascular endothelium','ExpressionParameters','Zero_Rate',0.0,1,1.0,1,NULL);
-INSERT INTO tab_container_parameter_rates VALUES(5015,'GENERAL','PROTEIN','Fraction expressed in blood cells','ExpressionParameters','Zero_Rate',0.0,1,1.0,1,NULL);
-INSERT INTO tab_container_parameter_rates VALUES(5015,'GENERAL','PROTEIN','Fraction expressed in endosomes','ExpressionParameters','Zero_Rate',0.0,1,1.0,1,NULL);
+INSERT INTO tab_container_parameter_rates VALUES(5015,'GENERAL','PROTEIN','Fraction expressed in blood cells','ExpressionParameters','One_Rate',0.0,1,1.0,1,NULL);

```

Comment:
One rate here. But it's not used since it's all read from the code. I am going to tackle that tomorrow. It won't be all good, but at least we won't duplicate all info like this